src/ode23s.jl

```diff
@@ -59,11 +59,11 @@ function show(io::IO, state::RosenbrockState)
 end
 
 
-function init{T}(ode::ExplicitODE{T},
```

Comment:
Hm, I don't know about this one, if it were `ivp::IVP` then fine, but `ivp::ExplicitODE` looks strange.  Shouldn't we pretend `ode is an instance of a different type `ExplicitODE`?